bin/rcm-release-data.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+
+here = sys.path[0]
+if here not in ('/usr/bin', '/bin', '/usr/local/bin'):
+    # git checkout
+    toplevel = os.path.dirname(here)
+    sys.path[0] = toplevel
+
+from distutils.sysconfig import get_python_lib
+
+sys.path.insert(1, os.path.join(get_python_lib(), "rcm_release"))
+
+from rcm_release import *
+
+DEFAULT_PDC_INSTANCE = 'https://pdc.engineering.redhat.com/rest_api/v1/'
```

Comment:
If we are expecting PDC to be on the system, we should just depend on the PDC client library, right? That way you will get the configured defaults and shortcuts for free.